src/main/resources/application.properties

```diff
@@ -14,4 +14,8 @@ aws.sourceBucket=sourceBucket
 aws.apiGatewayId=apiGatewayId
 aws.apiGatewayUsagePlanId=apiGatewayUsagePlanId
 
-#swagger-url= http://localhost:8080/swagger-ui/index.html
\ No newline at end of file
+#swagger-url= http://localhost:8080/swagger-ui/index.html
+
+# TODO: remove this secret
+jwt.secret-key=qwertyuiopasdfghjklzxcvbnm123456
```

Comment:
I'd remove this from version control all together and retrieve from the env variable. Once we commit it up, it's a pain to get rid of it, and it's public now!